week-8/Homework/mandatory/3-dog-photo-gallery/script.js

```diff
@@ -0,0 +1,34 @@
+function onLoad() {
+
+    var butElement = document.getElementById('newDogPhoto');
+    butElement.addEventListener("click", function addDogPhotoToDOM() {
+        fetch('https://dog.ceo/api/breeds/image/random')
+            .then(function(response) {
+                return response.json();
+
+
+            })
+            .then(function(obj) {
+                const listElement = document.createElement('ul');
```

Comment:
The original intention was to create only one `ul` element and append every new image to`li` as a new bullet point. Unfortunately, the description wasn't unequivocal.